pkg/sdk/roles_integration_test.go

```diff
@@ -0,0 +1,323 @@
+package sdk
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestInt_Roles(t *testing.T) {
+	client := testClient(t)
+	ctx := context.Background()
+
+	database, databaseCleanup := createDatabase(t, client)
+	t.Cleanup(databaseCleanup)
```

Comment:
Nice, only db cleanup, I like it <3

Reply:
Right, but not much gains here, because roles / users should have their own cleanup functions in the tests